wiki/Tournaments/LMS/Shift/1/en.md

```diff
@@ -0,0 +1,181 @@
+---
+tags:
+- LMS Shift
+- LMSShift
+---
+Taiko LMS: Shift Cup
+=============================
+
+![Taiko LMS: Shift Cup](banner.png)
+
+**Taiko LMS: Shift Cup** is a 3 on 3, 1v1 knock-out style osu!taiko tournament hosted by ![][flag_US][Backfire](https://osu.ppy.sh/users/263110). It is a new branch of the LMS Tournaments that are hosted yearly, which include LMS:SE and LMS:BG.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| ---: | :--- |
+| Registration Phase | 2018-04-11/2018-06-02 (09:00 UTC) |
+| Group Stage | 2018-06-09/2018-06-10 |
+| Round of 16 | 2018-06-16/2018-06-17 |
+| Quarterfinals | 2018-06-23/2018-07-24 |
+| Semifinals | 2018-06-30/2018-07-01 |
+| Finals | 2018-07-07/2018-07-08 |
+| Grand Finals | 2018-07-15 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| --- | :--- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Special profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | *to be decided* |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | *to be decided* |
+
+ *The remaining prizes are to be decided.*
+
+## Organization
+
+The Taiko LMS: Shift Cup is run by various osu!taiko community members.
+
+| Position | Members |
+| ------------ | -------------- |
+| Management and Organization | ![][flag_US] [Backfire](https://osu.ppy.sh/users/263110), ![][flag_CA] [FlamingRok](https://osu.ppy.sh/users/2015353), ![][flag_GB] [Lno](https://osu.ppy.sh/users/1900787) |
+| Map Selectors | ![][flag_US] [Backfire](https://osu.ppy.sh/users/263110), ![][flag_PT] [babysnakes](https://osu.ppy.sh/users/4669728), ![][flag_GB] [Lno](https://osu.ppy.sh/users/1900787) | |
+| Commentators | *To be decided* |
+| Streamer | ![][flag_DE] [Heamburger](https://osu.ppy.sh/users/4705120) |
+| Designer | ![][flag_FI] [Micro](https://osu.ppy.sh/users/4801080) |
+| Referees | ![][flag_AU] [cactusthegreat](https://osu.ppy.sh/users/5592409), ![][flag_GB] [Grimbow](https://osu.ppy.sh/users/1387762) |
+
+## Links
+
+- [Official Forums Thread](https://osu.ppy.sh/community/forums/topics/744163)
+- [Livestream](https://www.twitch.tv/osutaikolive)
+- [Registration](https://puu.sh/AmErh/c36eadf9e2.png)
```

Comment:
Did you screen capture this? This mouse cursor was captured
![image](https://user-images.githubusercontent.com/14264992/40096660-80e8e27a-5897-11e8-9536-72bcda090124.png)